proto/regen/data/v1alpha2/query.proto

```diff
@@ -11,10 +12,14 @@ option go_package = "github.com/regen-network/regen-ledger/x/data";
 // Query is the regen.data.v1alpha1 Query service
 service Query {
   // ByHash queries data based on its ContentHash.
-  rpc ByHash (QueryByHashRequest) returns (QueryByHashResponse);
+  rpc ByHash (QueryByHashRequest) returns (QueryByHashResponse) {
+    option (google.api.http).get = "/regen/data/v1alpha2/by-hash";
```

Comment:
why are we using `v1alpha2` here? If we stick to this, update the documentation above.